samples/meeting-live-coding-interview/csharp/MeetingLiveCoding/ClientApp/src/components/tab.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import * as microsoftTeams from "@microsoft/teams-js";
+import { Flex, Button, Text, Card, CardBody, CardHeader } from '@fluentui/react-northstar'
+import "./tab.css"
+import { IQuestionDetails } from '../types/question';
+
+const Tab = () => {
```

Comment:
I haven't run this locally, but based purely on the screenshots you shared, it looks like this component has overflows leading to scrollbars on the bottom and right. 
I'm not too worried about the right/vertical scroll bar, but can we update our styling to prevent the horizontal/bottom scroll bar?